### PR TITLE
ASDPLNG-328 Add deprecation message

### PR DESCRIPTION
**Pull request by bsper2**
Also did pdk update to clear pdk errors

### CODE REVIEW

**Comment by bsper2** [comment]
Only actual change is a warning in the README suggesting to use puppet-yum, the rest is `pdk update`.

Once this is added intend on archiving this repo.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,12 @@
 
 [![pdk-validate](https://github.com/ncsa/puppet-profile_additional_yumrepos/actions/workflows/pdk-validate.yml/badge.svg)](https://github.com/ncsa/puppet-profile_additional_yumrepos/actions/workflows/pdk-validate.yml)
  [![yamllint](https://github.com/ncsa/puppet-profile_additional_yumrepos/actions/workflows/yamllint.yml/badge.svg)](https://github.com/ncsa/puppet-profile_additional_yumrepos/actions/workflows/yamllint.yml)
+
  
+> :warning: This module is no longer recommended for configuring yum repos or yum proxy.
```
Comment (billglick) [inline README.md:3]: I think we want to make this slightly more bold/obvious that it is deprecated. The warning alone doesn't quite stand out enough to me when I view the README.md on github.

I saw we had done something similar to the following in `ncsa/puppet-ssh`:

```suggestion
# DEPRECATED

> :warning: This module is no longer recommended for configuring yum repos or yum proxy.
```

**Review by billglick — COMMENTED** [review]
Generally seems good, but I think we should make a big heading at the top that shows **deprecated** slightly more obvious.

File: README.md (hunk 1 above)
Comment (bsper2) [reply README.md:3]: Ok, I moved the Deprecated message to the top and made it a top level heading.

**Review by billglick — APPROVED** [review]
Looks good

**Pull request merged**